Rakefile

```diff
@@ -145,16 +145,10 @@ namespace :docker do
 end
 
 def environment_vars(with_coverage = true)
-  environment = travis_environment_variables
+  environment = []
   environment.concat(['-e', 'DO_COVERAGE=1']) if with_coverage && ENV['CIRCLECI']
   environment.concat(['-e', 'EAGER_LOAD=1'])
   environment.concat(['-e', "TEST_SUITE='#{ENV['TEST_SUITE']}'"])
   environment
 end
 
```

Comment:
Rubocop doesn-t like this line :unamused:

Reply:
no need to 'request changes' if the tests are failing

Reply:
nut I was so nice to tell you even without not need for it :roll_eyes:

Reply:
I'm not saying you should not complain - but the 'request changes' tends to stick in PRs :)